reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -331,6 +340,40 @@ class StaticHtmlReporter : TableReporter() {
                 }
             }
 
+    private fun createEvaluatorTable(evaluatorErrors: List<Error>) =
+            buildString {
+                append("<h2><a id=\"license-check-results\"></a>License Check Results</h2>")
+
+                if (evaluatorErrors.isEmpty()) {
+                    append("The evaluator did not find any issues!")
```

Comment:
Is mentioning the "evaluator" here too technical? Should we maybe just say "No issues found." instead?

Reply:
Done.